spyder/app/mainwindow.py

```diff
@@ -1182,6 +1181,12 @@ def create_edit_action(text, tr_text, icon):
         self.run = Run(self, configuration=CONF)
         self.register_plugin(self.run)
 
+        # Appearance plugin
+        CONF.set('appearance', 'css_path', css_path)

```

Comment:
Why is this an option of `appearance` and not of `help`? I say it because `css_path` is a path inside Help.

Reply:
Fixed